src/envoy/http/authn/BUILD

```diff
@@ -118,6 +118,17 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
+    name = "authn_utils_test",
+    srcs = ["authn_utils_test.cc"],
+    repository = "@envoy",
+    deps = [
+        ":authenticator",
```

Comment:
authn_util_lib

Reply:
#include "src/envoy/http/authn/authenticator_base.h" is required in authn_utils_test.cc and it is in ":authenticator"

Reply:
Why do you need that?
Anyway, if they have such a closed coupling, then maybe remove auth_utils lib, and add those files into authenticator directly.

Reply:
I can't see why you need authentication_base.h for auth_utils_test still. It's not a big deal, but would be nice to make sure you include only what you need.

Reply:
Done: authentication_base.h removed.

Reply:
[optional] then you can break the library again :)